server/core/src/interval.rs

```diff
@@ -56,17 +56,26 @@ impl IntervalActor {
     #[allow(clippy::result_unit_err)]
     pub fn start_online_backup(
         server: &'static QueryServerReadV1,
-        cfg: &OnlineBackup,
+        online_backup_config: &OnlineBackup,
         mut rx: broadcast::Receiver<CoreAction>,
     ) -> Result<tokio::task::JoinHandle<()>, ()> {
-        let outpath = cfg.path.to_owned();
-        let versions = cfg.versions;
-
+        let outpath = online_backup_config.path.to_owned();
+        let versions = online_backup_config.versions;
+        let crono_expr = online_backup_config.schedule.as_str().to_string();
+        let mut crono_expr_values = crono_expr.split_ascii_whitespace().collect::<Vec<&str>>();
+        if crono_expr_values.len() == 5 {
+            // we add a 0 element at the beginning to simulate the standard crono syntax which always runs
+            // commands at seconds 00
+            crono_expr_values.insert(0, "0");
+            crono_expr_values.push("*");
+        }
+        let crono_expression_schedule = crono_expr_values.join(" ");
```

Comment:
Since we're automatically rewriting a configuration item here, can you please add a `debug` statement to show the "supplied" vs "parsed" output? "don't surprise the users" is important. 😄

Reply:
oh crap I used info, is that alright too?

Reply:
Yeah, given it's only on startup that'll be fine. Thanks.